easybuild/easyconfigs/j/JAGS/JAGS-4.3.1-foss-2022a.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'JAGS'
+version = '4.3.1'
+
+homepage = 'http://mcmc-jags.sourceforge.net/'
+description = """JAGS is Just Another Gibbs Sampler.  It is a program for analysis
+ of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = [
+    ('https://sourceforge.net/projects/mcmc-%(namelower)s/files/%(name)s/%(version_major)s.x/Source/', 'download')
+]
+sources = [SOURCE_TAR_GZ]
+checksums = ['f9258355b5e9eb13bd33c5fa720f0cbebacea7d0a4a42b71b0fb14501ee14229']
+
+configopts = ' --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s', 'libexec/%(namelower)s-terminal', 'lib/libjags.%s' % SHLIB_EXT],
+    'dirs': [],
+}
+
```

Comment:
@sassy-crick don't suppose you feel like adding `sanity_check_commands`?

Reply:
sanity check added in 6ff63ca6f3